project-progression-policy.md

```diff
@@ -62,22 +61,17 @@ The TAC will determine the appropriate initial stage for the project.
 The project can apply for a different stage via the review process.
 
 Once a project passes Technical Review, the TAC will inform the Governing
-Board that the submission proposal and Technical Charter are ready for the
-next step.
+Board of the submission proposal.
 
 #### Legal Submission
 
+After a succesfull TAC vote, the project is ready for a legal review. The legal review is initated by the project filling out [this form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform?fbzx=4351560609072672295). The LF project onboarding team will create a draft Technical Charter, a draft Project Contribution Agreement (if needed), and a draft Series Agreement Draft.
```

Comment:
```suggestion
After a succesful TAC vote, the project is ready for a legal review. The legal review is initated by the project filling out [this form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform?fbzx=4351560609072672295). The LF project onboarding team will create a draft Technical Charter, a draft Project Contribution Agreement (if needed), and a draft Series Agreement.
```

Do you know if that is a fairly durable link or is there a higher level link somewhere in LF?

Is the Series Agreement also "if needed" or is it always needed?

Reply:
This is pretty durable and is the only link I am aware of. Yep series agreement is always needed/created.